pkg/virt-handler/options.go

```diff
@@ -61,24 +63,7 @@ func virtualMachineOptionsWithNetworkConfig(
 		return options, nil
 	}
 
-	domainAttachmentByPluginName := make(map[string]string)
-	networkBindings := clusterConfig.GetNetworkBindings()
-	if networkBindings != nil {
-		for name, binding := range networkBindings {
-			if binding.DomainAttachmentType != "" {
-				domainAttachmentByPluginName[name] = string(binding.DomainAttachmentType)
-			}
-		}
-	}
-
-	domainAttachmentByInterfaceName := make(map[string]string)
-	for _, iface := range vmiSpecIfaces {
-		if iface.Binding != nil {
-			domainAttachmentByInterfaceName[iface.Name] = domainAttachmentByPluginName[iface.Binding.Name]
-		}
-	}
-
-	options.InterfaceDomainAttachment = domainAttachmentByInterfaceName
+	options.InterfaceDomainAttachment = domainspec.DomainAttachmentByInterfaceName(vmiSpecIfaces, clusterConfig.GetNetworkBindings())
```

Comment:
This is changed in the wrong commit.
In any case, it now should be clear that there is no need to wrap this operation, this can be directly called at the callers that need it.

Reply:
done, still in this commit. I don't think it worth the effort moving it to the previous commit.